src/VisualStudio/Setup/AssemblyRedirects.cs

```diff
@@ -36,7 +36,7 @@
 
 [assembly: ProvideCodeBase(CodeBase = "$PackageFolder$\\System.AppContext.dll")]
```

Comment:
These are still here for Dev14 cases where we don't have the PortableFacades package installed into VS?